### DISCUSSION TITLE
How to make Qutebrowser do not switch to new tab when opening new links/tabs?

### DISCUSSION THREAD

**Discussion by uwla** [qutebrowser/qutebrowser#6180] [Q&A] [answered]
When I click on a link to open it on a new tab QuteBrowser automatically switches to that tab. The same happens if I open a new tab from a command.  Is there are a way to disabe that? So QuteBrowser nevers switches to a new tab, regardless if it was opened from the command or from clicking on a link

**Comment by The-Compiler** [comment, marked as answer]
For links, see the `tabs.background` setting (but also see #3231). For external commands, qutebrowser by default opens a new window. I'm guessing you set `new_instance_open_target` to `tab` - set it to `tab-bg` instead.

**Reply by uwla** [reply]
Thank you very much!
